src/utils/general.utils.ts

```diff
@@ -325,14 +325,18 @@ export const isTestnetChain = (chainId: string) => {
 
 export const areTokenAddressesEqual = (address1: string, address2: string): boolean => {
     if (address1.toLowerCase() === '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'.toLocaleLowerCase())
-        address1 = '0x0000000000000000000000000000000000000000'
+        address1 = ethers.constants.AddressZero
     if (address2.toLowerCase() === '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'.toLocaleLowerCase())
-        address2 = '0x0000000000000000000000000000000000000000'
+        address2 = ethers.constants.AddressZero
```

Comment:
nit: actually i slightly prefer the previous one because it's one less dependency on ethers and pretty self-evident

Reply:
makes sense!